test/ServiceBus.Tests/SlowConsumingTests/EHSlowConsumingTests.cs

```diff
@@ -109,7 +71,7 @@ public EHSlowConsumingTests(Fixture fixture)
             seed = new Random();
         }
 
-        [SkippableFact, TestCategory("Functional")]
+        [Fact, TestCategory("Functional")]
```

Comment:
this test can be 30 sec long. a bit too long for BVT. and we are running both functionals and bvt in Jenkins now.